plotsearch/serializers.py

```diff
@@ -469,3 +471,17 @@ def create(self, validated_data):
             plot_search.save()
 
         return plot_search
+
+
+class FavouriteTargetSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = FavouriteTarget
+        fields = ("plot_search_target",)
+
+
+class FavouriteSerializer(serializers.ModelSerializer):
```

Comment:
Shouldn't this have a overwritten create / update functions? Django fails to create or update nested serializers by default.